corehq/apps/locations/fixtures.py

```diff
@@ -227,25 +247,27 @@ def get_location_fixture_queryset(user):
     if user_locations.query.is_empty():
         return user_locations
 
+    return _location_queryset_helper(user.domain, list(user_locations.order_by().values_list("id", flat=True)))
+
+
+def _location_queryset_helper(domain, location_ids):
```

Comment:
I'd change this to `location_pks`

Reply:
https://github.com/dimagi/commcare-hq/pull/21349/commits/bcd47f7f629f9c4e97a2b14fa3e79c1828419a73